tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# mbed-build.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2019, ARM Limited, All Rights Reserved
+#
+# Purpose
+#
+# To run test builds of the mbed OS module for all specified targets.
+# Usage: mbed-build <folder to import to> <optinally string of space seperated targets>
+
+set -eu
+
+OUT_OF_SOURCE_DIR=$1
+MBED_APP='mbed-os-example-tls'
+REMOTE_URL="git@github.com:ARMmbed"
```

Comment:
Perhaps, but then it might collide with the current design of optionally giving list of targets. I wouldn't want to complicate the script with different types of input parameter checking